Github Issue Reader/ViewController.swift

```diff
@@ -55,8 +92,14 @@ class ViewController: UIViewController {
     func setupUI() {
         
         submitButton.titleLabel?.text = "Submit"
+        containerView.addArrangedSubview(gitLabel)
+        containerView.addArrangedSubview(gitSubLabel)
+        containerView.setCustomSpacing(20, after: gitSubLabel)
```

Comment:
Please extract this constant to a variable that describes what it's for / what it does.